MAINTAINERS.yml

```diff
@@ -393,14 +365,62 @@ Bluetooth Host:
     - sjanc
     - theob-pro
   files:
+    - doc/connectivity/bluetooth/
+    - include/zephyr/bluetooth/
+    - samples/bluetooth/
+    - subsys/bluetooth/common/
+    - subsys/bluetooth/crypto/
     - subsys/bluetooth/host/
+    - subsys/bluetooth/lib/
     - subsys/bluetooth/services/
     - subsys/bluetooth/shell/
-    - tests/bluetooth/host*/
-    - tests/bsim/bluetooth/host/
+    - subsys/bluetooth/CMakeLists.txt
+    - subsys/bluetooth/Kconfig*
+    - tests/bluetooth/
+    - tests/bsim/bluetooth/
   files-exclude:
```

Comment:
I think it can get tricky with this long exclusion list. It also shows that, may be, it will be better to keep controller and host into their own directories under Bluetooth.

Something like:
\bluetooth\common
\bluetooth\controller
\bluetooth\host
\bluetooth\mesh

Reply:
I agree. We also have a lot of common files that are not in the `common` directory. 

I think we can probably use the exclusion list as a checklist as to what needs/should be moved around to provide an overall more fragmented structure.

Reply:
This could be done later to streamline the file structure of Bluetooth code base.